cluster/examples/kubernetes/ceph/upgrade-from-v1.2-apply.yaml

```diff
@@ -0,0 +1,88 @@
+kind: ClusterRole
```

Comment:
Good on the upgrade!

Reply:
Make sure these apply items get added to the upgrade to 1.3 in the integration test please.